src/Discord.Net.Core/API/DiscordRestApiClient.cs

```diff
@@ -972,6 +972,34 @@ internal virtual void Dispose(bool disposing)
             }
             catch (HttpException ex) when (ex.StatusCode == HttpStatusCode.NotFound) { return null; }
         }
+        
+        //Relationships
+        public async Task<IReadOnlyCollection<Relationship>> GetRelationshipsAsync(RequestOptions options = null)
+        {
+            options = RequestOptions.CreateOrClone(options);
+            return await SendAsync<IReadOnlyCollection<Relationship>>("GET", () => "users/@me/relationships", new BucketIds(), options: options).ConfigureAwait(false);
+        }
+        public async Task AddFriendAsync(ulong userId, RequestOptions options = null)
```

Comment:
If the current relationship is IncomingRequest, this method accepts the request. If there is no relationship, it will send a friend request